### ISSUE TITLE
Make change / select dropdowns rather then buttons

### ISSUE THREAD

**Issue by arschmitz** [axemclion/perfjankie#20]
When using this for comparison it would be nice if you could easily switch test pages when your already within a test if the change and or select buttons were made to be drop downs with a list of the available pages this would be a lot nicer.

**Comment by axemclion** [comment]
This used to be the way it was done in an earlier version - http://nparashuram.com/bootstrap-perf. However, the number of pages grew like crazy :) I wanted to do an auto-complete type solution, but have not found time to do it.

**Comment by arschmitz** [comment]
Yeah i could see that becoming an issue pretty quickly autocomplete would be the way to go.

**Comment by axemclion** [comment]
I can get to this as soon as I finish the work to upgrade perfjankie to use the latest version of browser-perf. IF you have some time, a PR would be appretiated :+1:

**Comment by axemclion** [comment]
@arschmitz I looked at this, and found another interesting case that could come in the way - we could have a way to switch test pages, but not all test pages are run on all browsers. For example

`Test Page 1` is run on Firefox only. 
`Test Page 2` is run on Chrome only. 

Now, when `Test Page 2` is open, and we are viewing Chrome data, how would switching to `Test Page 1` work? Do we only show those pages that have run with the current browser?

**Comment by arschmitz** [comment]
I think in this case I would say if the test data for current browser is available show that otherwise show the first one which is available.
